### ISSUE TITLE
FormatterCLI and FormatterLogfmt do not output newlines

### ISSUE THREAD

**Issue by tv42** [z0mbie42/rz-go#15]
Multiple lines of log with FormatterCLI are all on one line.

```
tv@dark ~/z$ cat go.mod
module m

go 1.12

require github.com/bloom42/rz-go/v2 v2.6.0
tv@dark ~/z$ cat simple.go
package main

import (
	"github.com/bloom42/rz-go/v2"
)

func main() {
	l := rz.New(
		rz.Formatter(rz.FormatterCLI()),
	)
	l.Info("info from logger", rz.String("hello", "world"), rz.Caller(true))
}
tv@dark ~/z$ go run simple.go
✔ info from logger caller=/home/tv/z/simple.go:11 hello=worldtv@dark ~/z$
tv@dark ~/z$
tv@dark ~/z$
tv@dark ~/z$
tv@dark ~/z$ go run simple.go |od -a
0000000 esc   [   3   6   m   b  fs dc4  sp esc   [   0   m   i   n   f
0000020   o  sp   f   r   o   m  sp   l   o   g   g   e   r  sp esc   [
0000040   3   6   m   c   a   l   l   e   r esc   [   0   m   =   /   h
0000060   o   m   e   /   t   v   /   z   /   s   i   m   p   l   e   .
0000100   g   o   :   1   1  sp esc   [   3   6   m   h   e   l   l   o
0000120 esc   [   0   m   =   w   o   r   l   d
0000132
tv@dark ~/z$
```

**Comment by tv42** [comment]
FormatterLogfmt too:

```
tv@dark ~/z$ cat simple.go
package main

import (
	"github.com/bloom42/rz-go/v2"
)

func main() {
	l := rz.New(
		rz.Formatter(rz.FormatterLogfmt()),
	)
	l.Info("info from logger", rz.String("hello", "world"), rz.Caller(true))
}
tv@dark ~/z$ go run simple.go
 caller=/home/tv/z/simple.go:11 hello=world level=info message="info from logger" timestamp=2019-06-24T23:11:28Ztv@dark ~/z$
tv@dark ~/z$
tv@dark ~/z$
tv@dark ~/z$ go run simple.go |od -a
0000000  sp   c   a   l   l   e   r   =   /   h   o   m   e   /   t   v
0000020   /   z   /   s   i   m   p   l   e   .   g   o   :   1   1  sp
0000040   h   e   l   l   o   =   w   o   r   l   d  sp   l   e   v   e
0000060   l   =   i   n   f   o  sp   m   e   s   s   a   g   e   =   "
0000100   i   n   f   o  sp   f   r   o   m  sp   l   o   g   g   e   r
0000120   "  sp   t   i   m   e   s   t   a   m   p   =   2   0   1   9
0000140   -   0   6   -   2   4   T   2   3   :   1   1   :   3   2   Z
0000160
tv@dark ~/z$
```

**Comment by z0mbie42** [comment]
Hi, thank you for the report!

Fixed in  ae7d5a7
